libbeat/docs/breaking.asciidoc

```diff
@@ -25,10 +25,50 @@ See the following topics for a description of breaking changes:
 This section discusses the main changes that you should be aware of if you
 upgrade the Beats to version 7.0. {see-relnotes}
 
+==== Disable HTML escaping
```

Comment:
```suggestion
==== HTML escaping is disabled by default
```

Reply:
Added a few words to make it clear that you are describing a behavior and not telling users to do something.